src/Core/Commit.js

```diff
@@ -220,13 +228,13 @@ function commitOwner (element) {
 			case SharedElementPortal:
 				break
 			default:
-				element.owner = getElementBoundary(element, SharedLinkedNext).owner
+				element.owner = getElementBoundary(element, 'prev').owner
```

Comment:
What's the rationale for removing the enum reference?

Reply:
I'm not sure there was one but now that i think about it the enum reference might be more delightful.